tools/ci/run_unit_tests.sh

```diff
@@ -19,5 +19,5 @@ sudo chown -R $USER test-results
 # circleCI can be particularly slow sometimes since some time around 2021-06
 export OPENQA_TEST_TIMEOUT_SCALE_CI=3
 export EXTRA_PROVE_ARGS="-v"
-export OPENQA_FULLSTACK_TEMP_DIR=$PWD/test-results/fullstack
+export OPENQA_FULLSTACK_TEMP_DIR=$PWD/test-results/fullstack-$target
```

Comment:
I think the target is unique per circleci job anyway.
I think we would need the test name, as we use already in `COVERDB_SUFFIX`.
Why not reuse this?

Reply:
@perlpunk `retry` re-runs everything. The result would be the same:

    RETRY=${RETRY} HOOK=./tools/delete-artifacts-folder timeout -s SIGINT -k 5 -v ${TIMEOUT_RETRIES} tools/retry prove ${PROVE_LIB_ARGS} ${PROVE_ARGS}

Reply:
Oh, but indeed I overlooked the `unstable` case which is special in that it uses multiple coverdb prefixes within the target!